manifest.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "session-client",
+  "vendor": "vtex",
+  "version": "0.2.1",
```

Comment:
Why 0.2.1? 🤔

Reply:
I started from `graphql-example` app which had this version 🤷  haha